packages/database_v2/migrations/default/1631657122677_init/up.sql

```diff
@@ -0,0 +1,316 @@
+    token_uri TEXT NOT NULL,
+    -- Represents `ipfs://` URL for this asset. It is present if it was possible
+    -- to derive it from `token_uri`. E.g. If `token_uri` is IPFS gateway URL
+    -- https://ipfs.io/ipfs/Qm...Hash/file/path this field will be derived to
+    -- ipfs://Qm...Hash/file/path.
+    ipfs_url TEXT,
+    -- Cryptographic identifier of the content under `token_uri`. Will be only
+    -- present in `Lined` state.
+    content_cid TEXT,
+    status nft_asset_status NOT NULL,
+    status_text TEXT NOT NULL,
+
+    updated_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    inserted_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (token_uri)
+);
+
+-- Once `token_asset` is fetched and parsed as per ERC-721 metadata schema
+-- all mandatory fields will be stored in this table.
+CREATE TABLE IF NOT EXISTS nft_metadata (
+    -- Cryptographic identifier of the content this metadata was parsed from.
+    content_cid TEXT NOT NULL,
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    image_uri TEXT NOT NULL,
+
+    updated_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    inserted_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (content_cid)
+);
+
+-- Once nft asset is fetched and parsed all the ecountered URLs are saved
+-- as `resources` and concurrently archived. In the process resource state is
+-- updated as per `resource_status`.
+CREATE TABLE IF NOT EXISTS resource (
+    -- Unique Resource Identifier (URI) of this resource.
+    uri TEXT NOT NULL,
+    -- Current status of the resource.
+    status resource_status NOT NULL,
+    status_text TEXT,
+    -- Represents `ipfs://` URL for this asset. It is present if it was possible
+    -- to derive it from `token_uri`. E.g. If `token_uri` is IPFS gateway URL
+    -- https://ipfs.io/ipfs/Qm...Hash/file/path this field will be derived to
+    -- ipfs://Qm...Hash/file/path.
+    ipfs_url TEXT,
+    -- Cryptographic identifier of the content under `uri`. Will be only
+    -- present in `Lined` state.
+    content_cid TEXT,
+
+    updated_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    inserted_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (uri)
+);
+
+-- nft assets according to ERC-721 metadata spec must link to an `image`,
+-- however in practice various contracts also tend to include links to
+-- more resources in metadata JSON. This table maps `content_id` of the
+-- `nft_asset` / `nft_metadata` to those unspecified resources.
+CREATE TABLE IF NOT EXISTS other_nft_resources (
+    content_cid TEXT NOT NULL,
+    resource_uri TEXT NOT NULL,
+
+    updated_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    inserted_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (content_cid, resource_uri)
+);
+
+CREATE TABLE IF NOT EXISTS content (
+    cid TEXT NOT NULL,
+    dag_size INTEGER,
+
+    updated_at TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Does this differ from ours?:

```suggestion
    updated_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL,
```

Reply:
I've converted every date to explicitly specify 'utc' so there's not any issues with timezones, schemas will match now